crates/polars-ops/src/series/ops/round.rs

```diff
@@ -37,6 +37,43 @@ pub trait RoundSeries: SeriesSealed {
         polars_bail!(opq = round, s.dtype());
     }
 
+    fn round_sig_figs(&self, significant_figures: u32) -> PolarsResult<Series> {
+        let s = self.as_series();
+        let significant_figures = if significant_figures == 0 {
```

Comment:
Should I raise an exception of 0 is specified? How should this be done? Is it polars bail?

Reply:
I have done this.